tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
Does that mean that other parts of this file store their tests in the wrong place? Should `pep-420` be elsewhere?